app/menus/CollectionMenu.js

```diff
@@ -39,32 +68,47 @@ class CollectionMenu extends Component {
   };
 
   render() {
-    const { collection, label, onOpen, onClose, onImport } = this.props;
+    const { collection, label, onOpen, onClose } = this.props;
     const { allowDelete } = collection;
 
     return (
-      <DropdownMenu
-        label={label || <MoreIcon />}
-        onOpen={onOpen}
-        onClose={onClose}
-      >
-        {collection && (
-          <Flex column>
-            <DropdownMenuItem onClick={this.onNewDocument}>
-              New document
-            </DropdownMenuItem>
-            <DropdownMenuItem onClick={onImport}>
-              Import document
-            </DropdownMenuItem>
-            <DropdownMenuItem onClick={this.onEdit}>Edit…</DropdownMenuItem>
-          </Flex>
-        )}
-        {allowDelete && (
-          <DropdownMenuItem onClick={this.onDelete}>Delete…</DropdownMenuItem>
-        )}
-      </DropdownMenu>
+      <span>
```

Comment:
Ideally we'd use a React 16 array response here, but we need to update the flow definitions for that - I started doing this but it's actually quite a big task to update Flow which we should tackle separately.